cmd/vcert/flags.go

```diff
@@ -517,6 +529,101 @@ var (
+
+	flagKeyId = &cli.StringFlag{
+		Name:        "id",
+		Usage:       "The identifier of the requested certificate (usually used to determine ownership).",
+		Destination: &flags.sshCertKeyId,
+	}
+	flagObjectName = &cli.StringFlag{
+		Name:        "object-name",
+		Usage:       "The friendly name for the certificate object. If ObjectName is not specified, then KeyID parameter is used.",
+		Destination: &flags.sshCertObjectName,
+	}
+	flagDestinationAddress = &cli.StringFlag{
+		Name: "destination-address",
+		Usage: "The address (FQDN/hostname/IP/CIDR) of the destination host where the certificate will be used to authenticate to." +
+			"This is applicable for client certificates and used for reporting/auditing only",
+		Destination: &flags.sshCertDestAddr,
+	}
+	flagValidityHours = &cli.IntFlag{
+		Name:        "valid-hours",
+		Usage:       "How much time the requester wants to have the certificate valid, the format is hours. ",
+		Destination: &flags.sshCertValidHours,
+	}
+
+	flagSshCertCa = &cli.StringFlag{
+		Name:        "template",
+		Usage:       "the certificate issuing template that will be used.",
+		Destination: &flags.sshCertTemplate,
+	}
+
+	flagSshPubKey = &cli.StringFlag{
+		Name:        "public-key",
+		Usage:       "if user will provide a public key, local or service generated",
+		Value:       "service",
+		Destination: &flags.sshCertPubKey,
+	}
+
+	flagSshKeySize = &cli.IntFlag{
+		Name:        "key-size",
+		Usage:       "key size bits, they will be used for creating keypair on case public-key:local",
+		Destination: &flags.sshCertKeySize,
+	}
+
+	flagSshPassPhrase = &cli.StringFlag{
+		Name:        "key-passphrase",
+		Usage:       "passphrase for encrypting the private key",
+		Destination: &flags.sshCertKeyPassphrase,
+	}
+
+	flagPolicyDN = &cli.StringFlag{
+		Name:        "folder",
+		Usage:       "The DN of the policy folder where the certificate object will be created. If this is not specified, then the policy folder specified on the certificate template will be used.",
+		Destination: &flags.sshCertFolder,
+	}
+
+	flagForceCommand = &cli.StringFlag{
+		Name:        "force-command",
+		Usage:       "The requested force command. Example: /usr/scripts/db_backup.sh",
+		Destination: &flags.sshCertForceCommand,
+	}
+
+	flagSourceAddresses = &cli.StringSliceFlag{
+		Name:  "source-address",
+		Usage: "The requested source addresses as list of IP/CIDR. Example: 192.168.1.1/24",
+	}
+
+	flagSshCertPickupId = &cli.StringFlag{
+		Name:        "pickup-id",
+		Usage:       "the SSH Certificate DN",
+		Destination: &flags.sshCertPickupId,
+	}
+
+	flagSshCertGuid = &cli.StringFlag{
+		Name:        "guid",
+		Usage:       "A value that uniquely identifies the certificate request.",
+		Destination: &flags.sshCertGuid,
+	}
+
+	flagSshPrivKeyPassphrase = &cli.StringFlag{
+		Name: "pk-passphrase",
+		Usage: "The passphrase which will be used to wrap the generated private key before it is returned in the API response." +
```

Comment:
Why can't we use `key-passphrase` for this use case?

Reply:
yes actually this is duplicated, key-passphrase is the one that we use